agent/consul/client_test.go

```diff
@@ -219,15 +224,13 @@ func TestClient_RPC_Pool(t *testing.T) {
 }
 
 func TestClient_RPC_ConsulServerPing(t *testing.T) {
+	t.Parallel()
 	var servers []*Server
 	var serverDirs []string
 	const numServers = 5
 
-	for n := numServers; n > 0; n-- {
-		var bootstrap bool
-		if n == numServers {
-			bootstrap = true
-		}
+	for n := 0; n < numServers; n++ {
+		bootstrap := n == 0
```

Comment:
previously this would set bootstrap to true when n=5, now it does when n=0. Not familiar enough with this part of the code, does this change the semantics of the test at all?